### PR TITLE
OSD-4181: Unify PHONY targets

### PR DESCRIPTION
**Pull request by lisa**
There were extra hyphens.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -59,13 +59,13 @@ delete-account:
 test-account-creation: create-account delete-account
 
 # Create account claim namespace
-.PHONY: create-account-claim-namespace
+.PHONY: create-accountclaim-namespace
```
Comment (fahlmant) [inline Makefile:62]: Line 284 uses `create-account-claim-namespace` and need to be updated

Comment (lisa) [reply Makefile:62]: Fixed. Good catch.

**Comment by jharrington22** [comment]
Thanks @lisa ! looks good to me, squash first and I'll merge.

**Comment by lisa** [comment]
> Thanks @lisa ! looks good to me, squash first and I'll merge.

Squashed.

**Pull request merged**